test/testcafe/spec/TerminalReturnEmail_spec.js

```diff
@@ -1,4 +1,3 @@
-import IdentityPageObject from '../framework/page-objects/IdentityPageObject';
```

Comment:
looks like eslint doesn't catch it. mind want to follow up?

Reply:
yeah, I just found this. Will create JIRA